fdbserver/KeyValueStoreRocksDB.actor.cpp

```diff
@@ -222,6 +231,11 @@ ACTOR Future<Void> rocksDBMetricLogger(std::shared_ptr<rocksdb::Statistics> stat
 			ASSERT(db->GetIntProperty(property, &stat));
 			e.detail(name, stat);
 		}
+
+		if (SERVER_KNOBS->ROCKSDB_PERFCONTEXT_ENABLE) {
+			TraceEvent("RocksDBPerfMetrics").setMaxFieldLength(2048).detail("Metrics", rocksdb_perfcontext_report(perf, true /* Exclude zero counters */));
```

Comment:
Just curios, will this length(2048) be sufficient? Looks like they are around 100 attributes.

Reply:
most of them are 0 in my test. 
In case it's not enough, we can change it to bigger number.